lib/api/route/streams.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../core.dart';
+part 'streams.g.dart';
+
+/// https://zulip.com/api/get-stream-topics
+Future<GetTopicsResult> getStreamTopics(
+  ApiConnection connection, {
```

Comment:
nit: format the same as other endpoint methods:
```suggestion
Future<GetTopicsResult> getStreamTopics(ApiConnection connection, {
```